doc/articles/controls/GLCanvasElement.md

```diff
@@ -0,0 +1,201 @@
+---
+uid: Uno.Controls.GLCanvasElement
+---
+
+## Uno.WinUI.Graphics3D.GLCanvasElement
+
+> [!IMPORTANT]
+> This functionality is only available on WinUI and Skia Desktop (`netX.0-desktop`) targets that are running with Desktop OpenGL (not GLES) hardware acceleration. This is also not available on MacOS.
+
+`GLCanvasElement` is a `Grid` for drawing 3D graphics with OpenGL. This class comes as a part of the `Uno.WinUI.Graphics3D` package.
```

Comment:
Let's make individual packages for each 3D targets, so that we don't bring many dependencies when not needed. Something like `Uno.WinUI.Graphics3DGL`.

A few other pieces:
- We also need to document the fact that `Graphics3DGL` is a feature so we'll need to add it [here](https://github.com/unoplatform/uno/blob/master/src/Uno.Sdk/targets/Uno.Implicit.Packages.ProjectSystem.Desktop.targets) and [here](https://github.com/unoplatform/uno/blob/master/src/Uno.Sdk/UnoFeature.cs). 
- We also need to add a regression validation [here](https://github.com/unoplatform/uno/blob/fe14d62e2400576e9a7fc14c15ca693b725e2f53/src/SolutionTemplate/5.2/uno52blank/uno52blank/uno52blank.csproj#L34), likely by adding a new test line [here](https://github.com/unoplatform/uno/blob/fe14d62e2400576e9a7fc14c15ca693b725e2f53/build/test-scripts/run-net7-template-linux.ps1#L71).

Reply:
Can you add those to the TOC please?